File: Practica 2.2.1/Python/Especimen.java
public class Especimen extends Organismo {
	private Object _especie;
	private Object _raza;
	private Object _dieta;

	public void getEspecie() {
		return this._especie;
	}

	public void setEspecie(Object aEspecie) {
		this._especie = aEspecie;
	}

	public void getRaza() {
		return this._raza;
	}

	public void setRaza(Object aRaza) {
		this._raza = aRaza;
	}

	public void getDieta() {
		return this._dieta;
	}

	public void setDieta(Object aDieta) {
		this._dieta = aDieta;
	}
}

File: Practica 2.2.1/Python/Especimen.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
import Organismo
from typing import List

class Especimen(Organismo):
	def getEspecie(self):
		return self.___especie

	def setEspecie(self, aEspecie) -> None:
		self.___especie = aEspecie

	def getRaza(self):
		return self.___raza

	def setRaza(self, aRaza) -> None:
		self.___raza = aRaza

	def getDieta(self):
		return self.___dieta

	def setDieta(self, aDieta) -> None:
		self.___dieta = aDieta

	def __init__(self):
		self.___especie = None
		self.___raza = None
		self.___dieta = None

